modules/site-health/audit-database/class-perflabdbmetrics.php

```diff
@@ -0,0 +1,433 @@
+<?php
+/**
+ * Performance Lab database metrics computation module
+ *
+ * @package performance-lab
+ * @group audit-database
+ *
+ * @since 1.4.0
+ */
+
+/**
+ * Performance Lab database metrics class,
+ *
+ * @package performance-lab
+ * @group audit-database
+ *
+ * @since 1.4.0
+ */
+class PerflabDbMetrics {
+	/** First eligible db version, the advent of utfmb4 in WordPress databases.
+	 *
+	 * @var int
+	 */
+	public $first_compatible_db_version = 32814;
+	/** Latest eligible db version.
+	 * TODO: retest when new DB versions become available.
+	 *
+	 * @var int
+	 */
+	public $last_compatible_db_version = 53496;
+	/** OK to gather metrics when $wp_db_version is greater than $last_compatible_db_version.
+	 *
+	 * @var bool false if we should error out on newer version.
```

Comment:
Please leave out the description behind `@var` throughout the PR.
```suggestion
	 * @since n.e.x.t
	 * @var bool
```